sgt-back-end/index.js

```diff
@@ -0,0 +1,128 @@
+const pg = require('pg');
+const express = require('express');
+const app = express();
+const unexpectedError = { error: 'An unexpected error occurred' };
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+app.get('/api/grades', (req, res) => {
+  const sql = `
+        select *
+          from "grades"`;
+  db.query(sql)
+    .then(result => {
+      const grade = result.rows;
+      res.status(200).json(grade);
+    })
+    .catch(err => {
+      res.send(500).json(unexpectedError);
+      process.stderr.write(err);
+      process.exit(1);
```

Comment:
We don't want to exit the process, because that will kill the server. Instead just send the 500 response and print the error for debugging purposes. Be sure to fix this in all of your routes.